script/upgrades/COPWhitelist/deploy/COPWhitelist-00-Deploy-ChainlinkRelayers.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// solhint-disable contract-name-camelcase
+pragma solidity ^0.8.18;
+
+import { console } from "forge-std-next/console.sol";
+import { Script } from "script/utils/mento/Script.sol";
+import { Chain as ChainLib } from "script/utils/mento/Chain.sol";
+import { Contracts } from "script/utils/mento/Contracts.sol";
+
+import { ChainlinkRelayerFactory } from "mento-core-2.5.0/oracles/ChainlinkRelayerFactory.sol";
+import { ChainlinkRelayerFactoryProxy } from "mento-core-2.5.0/oracles/ChainlinkRelayerFactoryProxy.sol";
+import { ChainlinkRelayerFactoryProxyAdmin } from "mento-core-2.5.0/oracles/ChainlinkRelayerFactoryProxyAdmin.sol";
+import { IChainlinkRelayer } from "mento-core-2.5.0/interfaces/IChainlinkRelayer.sol";
+import { toRateFeedId, aggregators } from "script/utils/mento/Oracles.sol";
+
+contract cCOP_Deploy_ChainlinkRelayers is Script {
+  using Contracts for Contracts.Cache;
+  using { toRateFeedId } for string;
+
+  struct Relayer {
+    string rateFeed;
+    string rateFeedDescription;
+    uint256 maxTimestampSpread;
+    IChainlinkRelayer.ChainlinkAggregator[] aggregators;
+  }
+
+  Relayer[] private relayers = [
+    Relayer({
+      rateFeed: "relayed:CELOCOP",
+      rateFeedDescription: "CELO/COP (CELO/USD:USD/COP)",
+      maxTimestampSpread: 24 hours,
```

Comment:
Chainlink updated CELO/USD to also report every 5 minutes, so I think we could lower this now to 5 or 6 minutes

Reply:
Here's Chainlink's CELO/USD aggregator reporting every ~5mins now https://celoscan.io/address/0x5FB8bFAEBe2dd6dB2AF308D293ea25e607D3A922

Reply:
Just going to keep this as is for now.